src/Components/TopBar/styles.ts

```diff
@@ -3,14 +3,17 @@ import { StyleSheet } from "aphrodite";
 const styles = StyleSheet.create({
   topBar: {
     backgroundColor: "#a7c4a0",
-    height: "10vh",
+    height: "50px",
+    width: '100%',
     padding: "7px",
     boxSizing: "border-box",
     display: "flex",
     justifyContent: "space-between",
     alignItems: "center",
     boxShadow: "0px 10px 10px -8px gray",
     marginBottom: 20,
+    position: 'fixed',
+    zIndex: 5,
```

Comment:
is this zIndex necesary? otherwise remove it

Reply:
It is necessary for the TopBar to function